src/k4a_ros_device.cpp

```diff
@@ -692,692 +680,700 @@ k4a_result_t K4AROSDevice::fillPointCloud(const k4a::image& pointcloud_image, se
+        "Setting the time base using a k4a_image_t timestamp. This will result in a "
+        "larger uncertainty than setting the time base using the timestamp of a k4a_imu_sample_t sample. "
+        "Assuming the transmission delay to be "
+        << transmission_delay.toSec() * 1000.0 << " ms.");
+    start_time_ = ros::Time::now() - duration_since_device_startup - transmission_delay;
+  }
+  return start_time_ + duration_since_device_startup;
 }
 
 // Converts a k4a_imu_sample_t timestamp to a ros::Time object
-ros::Time K4AROSDevice::timestampToROS(const uint64_t & k4a_timestamp_us)
+ros::Time K4AROSDevice::timestampToROS(const uint64_t& k4a_timestamp_us)
 {
-    ros::Duration duration_since_device_startup(k4a_timestamp_us / 1e6);
+  ros::Duration duration_since_device_startup(k4a_timestamp_us / 1e6);
 
-    // Set the time base if it is not set yet.
-    if (start_time_.isZero())
-    {
-      const ros::Duration transmission_delay(0.005);
-      ROS_INFO_STREAM("Setting the time base using a k4a_imu_sample_t sample. "
-        "Assuming the transmission delay to be " << transmission_delay.toSec() * 1000.0 << " ms.");
-      start_time_ = ros::Time::now() - duration_since_device_startup - transmission_delay;
-    }
-    return start_time_ + duration_since_device_startup;
+  // Set the time base if it is not set yet.
+  if (start_time_.isZero())
+  {
+    const ros::Duration transmission_delay(0.005);
+    ROS_INFO_STREAM(
+        "Setting the time base using a k4a_imu_sample_t sample. "
+        "Assuming the transmission delay to be "
+        << transmission_delay.toSec() * 1000.0 << " ms.");
+    start_time_ = ros::Time::now() - duration_since_device_startup - transmission_delay;
+  }
+  return start_time_ + duration_since_device_startup;
 }
 
-void printTimestampDebugMessage(const std::string& name, const ros::Time & timestamp)
+void printTimestampDebugMessage(const std::string& name, const ros::Time& timestamp)
 {
-    ros::Duration lag = ros::Time::now() - timestamp;
-    static std::map<const std::string, std::pair<ros::Duration, ros::Duration>> map_min_max;
-    auto it = map_min_max.find(name);
-    if (it == map_min_max.end())
-    {
-        map_min_max[name] = std::make_pair(lag, lag);
-        it = map_min_max.find(name);
-    }
-    else
-    {
-        auto & min_lag = it->second.first;
-        auto & max_lag = it->second.second;
-        if (lag < min_lag)
-            min_lag = lag;
-        if (lag > max_lag)
-            max_lag = lag;
-    }
+  ros::Duration lag = ros::Time::now() - timestamp;
+  static std::map<const std::string, std::pair<ros::Duration, ros::Duration>> map_min_max;
+  auto it = map_min_max.find(name);
+  if (it == map_min_max.end())
+  {
+    map_min_max[name] = std::make_pair(lag, lag);
+    it = map_min_max.find(name);
+  }
+  else
+  {
+    auto& min_lag = it->second.first;
+    auto& max_lag = it->second.second;
+    if (lag < min_lag) min_lag = lag;
+    if (lag > max_lag) max_lag = lag;
```

Comment:
Can we fix this now and require `{ }` for all `if` statements?

Reply:
+100! Yes fixed, and looked through rest of code and couldn't find other instances. Maybe one day we can also set up a linter for these kinds of issues, but it's probably a bit out of scope. ;)